src/plugins/auto_batch/tests/functional/behavior/plugin/auto_batching_tests.hpp

```diff
@@ -1,47 +1,46 @@
-// Copyright (C) 2018-2022 Intel Corporation
+// Copyright (C) 2018-2023 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
is this file about auto-batch testing or about integration between HW plugin and AUTO_BATCH?
If it's the second case, we need to keep this file in behavior tests